safenode/src/client/api.rs

```diff
@@ -248,7 +248,7 @@ impl Client {
     // If `get_all_responses` is false, we return the first successful response that we get.
     async fn send_and_get_responses(
         &self,
-        nodes: HashSet<PeerId>,
+        nodes: Vec<PeerId>,
```

Comment:
Seems it could be useful to ensure we don't have duplicates here?
Probably handled in lower layers (I assume). But still?

Reply:
Btw, was talking with Chris about node/peer distinction.

We've been trying to keep a distinction bewtween nodes and peers (I know this wasn't changed in this PR, but since this line is modified here).

So when we have peer ids, we should call them peers.

When we talk of our node, then we should call it node.

Reply:
The result that we get from the `Swarm` is a `HashSet` which we then order it at the network layer. So the final value that we get here is ordered and contains unique peers. I thought preserving the order might be worth it.

Reply:
Yes, could absolutely be :+1: